### PR TITLE
Gatsby: Fix measure button hover

### PR DESCRIPTION
**Pull request by Vesely**

### CODE REVIEW

File: gatsby/src/components/measure-list/measure.module.scss [hunk 1]
```diff
@@ -33,7 +33,10 @@
   border-color: #2362a2;
   color: #2362a2;
 
-  &:hover {
-    background-color: #2362a2;
+  &:hover,
+  &:focus {
+    border-color: #254e80;
```
Comment (fidelman) [inline gatsby/src/components/measure-list/measure.module.scss:38]: maybe we could use variables for brand colors?

Comment (vitkolos) [reply gatsby/src/components/measure-list/measure.module.scss:38]: those variables are inside our design system (base/_variables)

Comment (Vesely) [reply gatsby/src/components/measure-list/measure.module.scss:38]: @vitkolos I am just fix hover effect, can you create new ticket for use variables in module styles?

Comment (vitkolos) [reply gatsby/src/components/measure-list/measure.module.scss:38]: it's there

Comment (vitkolos) [reply gatsby/src/components/measure-list/measure.module.scss:38]: https://trello.com/c/n8t31meC/79-setup-ds-variables-in-module-styles

**Review by kkoudelka — APPROVED** [review]

**Pull request merged**